stage1/init/common/pod.go

```diff
@@ -195,13 +195,35 @@ func SetJournalPermissions(p *stage1commontypes.Pod) error {
 		return fmt.Errorf("error creating journal dir: %v", err)
 	}
 
-	a, err := acl.Parse(fmt.Sprintf("g:%d:r-x,m:r-x", rktgid))
+	a, err := acl.InitACL()
 	if err != nil {
-		panic(fmt.Sprintf("error parsing ACL string: %v", err))
+		return err
+	}
+
+	if err := a.ParseACL(fmt.Sprintf("g:%d:r-x,m:r-x", rktgid)); err != nil {
+		return fmt.Errorf("error parsing ACL string: %v", err)
 	}
 	defer a.Free()
 
-	if err := a.SetFileDefault(journalPath); err != nil {
+	val, err := a.Valid()
+	if err != nil {
+		return err
+	}
+	if !val {
+		if err := a.AddBaseEntries(journalPath); err != nil {
+			return fmt.Errorf("error adding base ACL entries: %v", err)
+		}
+
+		val, err := a.Valid()
```

Comment:
I guess that the calls to `Valid` are some kind of error checking if previous call succeeded?